Decred Wallet/Assets.xcassets/backup_checkbox_default.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "backup_checkbox_default.png",
```

Comment:
I don't think we need an image for this. A simple view of equal width and height, border radius set to 50% and border color+width specified should suffice. On check, we can set the background color to the checked image and remove border color.